### ISSUE TITLE
Make outline clickable by default for the simple theme

### ISSUE THREAD

**Issue by m-kru** [andreasKroepelin/polylux#177]
By default, the simple theme has unclickable outline. Changing this behavior is not simple as user cannot simply call `outline` with custom arguments to change this behavior.

In my opinion, the outline should be clickable by default so that you can easily jump to slides using mouse. If you don't want to use this functionality you simply don't click. Right now, achieving clickable outline is impossible for a noob user.

Actually, I do not understand why the simple theme file configures the outline at all. The outline is easily configurable via the typst built-in `outline` function.

**Comment by andreasKroepelin** [comment]
Good idea. Here is some background on why there is custom overview handling at all:
https://polylux.dev/book/utils/sections.html
